src/App.js

```diff
@@ -14,9 +15,11 @@ function App() {
       <Route path = "/"element = {<Homepage/>}/> 
       <Route path = "/mock" element = {<Mockman />}/> 
       <Route path = "/videos" element = {<VideoListing />}/> 
+      {/* <Route path = "/videos/:videoId" element = {<VideoListing />}/>  */}
```

Comment:
You can remove commented code.

Reply:
I was trying to make single video page but will make in next functionality. So I kept it